templates/store.php

```diff
@@ -8,10 +8,15 @@
 
 if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
 
-$store_user   = dokan()->vendor->get( get_query_var( 'author' ) );
-$store_info   = $store_user->get_shop_info();
-$map_location = $store_user->get_location();
-$layout       = get_theme_mod( 'store_layout', 'left' );
+$store_user                = dokan()->vendor->get( get_query_var( 'author' ) );
+$store_info                = $store_user->get_shop_info();
+$map_location              = $store_user->get_location();
+$layout                    = get_theme_mod( 'store_layout', 'left' );
+$vendor_id                 = $store_user->id;
+$products_block_appearance = dokan_get_option( 'store_products_section', 'dokan_appearance' );
+$store_settings            = dokan_get_store_info( $vendor_id );
```

Comment:
Please recheck this `$store_settings ` value if we already got this from `$store_info`.
As behind the scene, this called the `get_shop_info()` function too.